README.rst

```diff
@@ -42,16 +42,32 @@ Initial setup and installation:
 -  copy sample local settings and configure for your environment::
    cp winthrop/local_settings.py.sample winthrop/local_settings.py
 
+- Create a new Solr cores with a basic configuration and managed schema,
+  using the core/collection names that youconfigured in local settings::
+
+    solr create -c winthrop -n basic_configs
```

Comment:
I am curious why the `-n` option as it (from what I can tell)  just specifies a config name? More to make sure I'm consistent on the DevOps end.